### ISSUE TITLE
Limits based on Sonarr or Radarr independently 

### ISSUE THREAD

**Issue by tvann2182** [petio-team/petio#392]
**Is your feature request related to a problem? Please describe.**
When under the user section there is only one option for request limits. This presents a problem when you only allow certain users to request so many tv episodes and movies at a time

**Describe the solution you'd like**
What i would like to see is a set request limit option for tv episodes and then a request limit for movies, this can be weekly monthly however you want to create it. 

**Describe alternatives you've considered**
Currently there are no other alternate ways besides either giving someone unlimited or a focused amount for both tv and movie releases. this can lead to extreme file additions and cause a drive to fill up without providing enough limiting on the user class

**Additional context**
Keep up the good work

**Comment by Drazzilb08** [comment]
To add to this. There for sure needs to be another setting to base upon a time frame. X movies per day/week/month and same for tv shows/episodes per day/week/month. Currently there is only x requests per week. The addition to increase the timeframe would make fine tuning very helpful.

**Comment by Fronix** [comment]
Yes, this is something that's hindering me from actually rolling out Petio. On top of this, when requesting a TV show, you should be able to choose what season/episodes you want to request.

Full configuration of expiration time of requests is also something I'd love. Not being able to control when request limits expire is annoying.

What would be even better is being able to create your own request limit filters. This would be useful to limit certain types of shows or movie requests. Maybe you want to limit the amount of anime shows, you could just create an anime filter that puts a limit on those types of request to X per w/e time you configure.

**Examples:**
TV Show limit filter based on Genre:
- Condition: Genre
- Value: Anime
- Limit: 20
- Expiration: 1w

Movie limit filter:
- Condition: Genre
- Value: Anime
- Limit: 10
- Expiration: 1w

And to make sure there's always a limit, a base configuration would exist to catch anything not matching any filters. That would have only the **Limit** and **Expiration**. This way, you can totally ignore limit filters if you don't want it.
